internal/transform/ledger.go

```diff
@@ -0,0 +1,127 @@
+package transform
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+//ConvertLedger converts a ledger from the history archive ingestion system into a form suitable for BigQuery
+func ConvertLedger(inputLedgerMeta xdr.LedgerCloseMeta) (LedgerOutput, error) {
+	ledger, ok := inputLedgerMeta.GetV0()
+	if !ok {
+		return LedgerOutput{}, errors.New("Could not access the version 0 information for the provided ledger")
```

Comment:
Preference for `fmt.Errorf` instead of `errors.New`, though it matters less when you have no formatting. I'd also make the error message a bit shorter, since the developer likely has context: "could not access v0 info for given ledger". It would be even better if you could print out the ledger's sequence number in the error, so the developer could more easily debug.